ArduPlane/Parameters.cpp

```diff
@@ -1270,6 +1270,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(systemid, "SID", 38, ParametersG2, AP_SystemID),
 #endif
     
+    // @Param: CLIMB_SLOPE_HGT
+    // @DisplayName: Climb slope mininum height
+    // @Description: Sets the minimum height above home at which the aircraft will apply a climb slope between waypoints. Below it, the aircraft will ascend immediately, and will only resume the requested trajectory upon reaching this height. This prevents unsafe behavior such as attempting to slowly gain altitude near obstacles. The default value of 25 m. ensures safe operations in most environments, but it can be adjusted based on specific terrain or operational needs.
```

Comment:
```suggestion
    // @Description: Sets the minimum height above home at which the aircraft will apply a climb slope between waypoints. Below it, the aircraft will ascend immediately, and will only resume the requested trajectory upon reaching this height. This prevents unsafe behavior such as attempting to slowly gain altitude near obstacles. The default value, ensures safe operations in most environments, but it can be adjusted based on specific terrain or operational needs.
```
Users can see the defualt in Mission Planner, including it in the text usualy just means we forget to update it if we change the defualt. People could also have loaded a custom defualt value in a custom build.